scoring.js

```diff
@@ -36,8 +36,42 @@ function calculateReturnScore(player) {
 }
 
 module.exports.calculateScore = function (player) {
-  return calculatePassingScore(player) +
-    calculateRushingScore(player) +
-    calculateReceivingScore(player) +
-    calculateReturnScore(player)
+  /* return calculatePassingScore(player) +
```

Comment:
Commenting out code is very useful when we need to keep it around for reference while working but always be sure to delete it before committing your changes. Once you're done writing the working code, the old commented out code is just clutter.